src/Engine/EmitMSIL/CodeGenIL.cs

```diff
@@ -110,7 +110,7 @@ internal CodeGenIL(IDictionary<string, IList> input, string filePath, ProtoCore.
             // 1. Create assembly builder (dynamic assembly)
             var asm = BuilderHelper.CreateAssemblyBuilder("DynamicAssembly", false, access);
             // 2. Create module builder
-            var mod = BuilderHelper.CreateDLLModuleBuilder(asm, "DynamicModule");
+            var mod = BuilderHelper.CreateDLLModuleBuilder(asm, "DynamicAssembly");
```

Comment:
?

Reply:
I'm not sure exactly whats going on locally - but if this is not named the same as the Dynamic Assembly file it ends up generating a separate module file that I cannot decompile, and the .dll fails to decompile.

Reply:
I was able to decompile the separately generated DynamicModule.dll but found DynamicAssembly.dll to be more or less empty. In any case, I was unclear what was the real difference between them and it might be that both need to be named the same as you're doing here.